presto-docs/src/main/sphinx/release/release-0.252.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.252
+=============
+
+**Highlights**
+==============
+* Add new spilling strategy to spill when a combination of user, system and recovable memory on a node exceeds ``max_total_memory_per_node``.
+* Add support for logging formatted prepared query.
+* Add support to coordinator endpoint ``/v1/info/state`` to return ``ACTIVE`` when the coordinator is not shutting down and the cluster has the minimum required workers.
```

Comment:
Is this highlight worthy ? 

For me highlights is something significantly new. These things are listed in the section just below - so we don't need to highlight everything.